tests/src/test_cases/test_arc.c

```diff
@@ -166,7 +166,7 @@ void test_arc_click_sustained_from_start_to_end_does_not_set_value_to_max(void)
 
     lv_obj_set_size(arc, 100, 100);
     lv_obj_center(arc);
-    lv_obj_add_event(arc, dummy_event_cb, LV_EVENT_PRESSED, NULL);
+    lv_obj_add_event_cb(arc, dummy_event_cb, LV_EVENT_PRESSED, NULL);
```

Comment:
This operation looked the same to me, only function name was updated between v8 and v9, right?

Reply:
Yep!